lib/src/tagging/hash.ts

```diff
@@ -0,0 +1,17 @@
+import { createHash } from "crypto";
+import { ContractDefinition } from "../models/definitions";
+
+export function hashContractDefinition(
+  contractDefinition: ContractDefinition
+): string {
+  const contractDefinitionString = JSON.stringify(contractDefinition).replace(
+    /\s/g,
```

Comment:
Why is this necessary?

Reply:
Don't know if this is really necessary, but read that `JSON.stringify` could add unwanted whitespace - so that would make it formatting independent.

Reply:
I'd prefer to remove it.